pages/requests/create-replacement.tsx

```diff
@@ -0,0 +1,184 @@
+import Layout from '@components/admin/Layout'; // Layout component
+import { Text, Box, Flex, Stack, Button, GridItem, Input } from '@chakra-ui/react'; // Chakra UI
+import { useState } from 'react'; // React
+import PermitHolderInformationForm from '@components/admin/requests/forms/PermitHolderInformationForm'; //Permit holder information form
+import { PermitHolderInformation } from '@tools/components/admin/requests/forms/types'; //Permit holder information type
```

Comment:
Combine the imports from `@tools/components/admin/requests/forms/types` into one import